aws-ebs-csi-driver/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ebs-csi-controller-sa
+  namespace: kube-system
```

Comment:
Could you add optional annotations to the ServiceAccount so we can use IAM role for service accounts via helm please?  See https://docs.aws.amazon.com/eks/latest/userguide/specify-service-account-role.html

Reply:
Sure